src/main/java/appeng/block/misc/BlockInterface.java

```diff
@@ -52,6 +57,26 @@ public BlockInterface()
 	}
 
 	@Override
+	protected IProperty[] getAEStates()
+	{
+		return new IProperty[] { AE_BLOCK_FORWARD, AE_BLOCK_UP, OMNIDIRECTIONAL };
```

Comment:
Should something like `Stream.concat(super.getAEStates(), new IProperty[] { OMNIDIRECTIONAL }).toArray(IProperty[]::new);` not work?